ghproxy/ghmetrics/ghmetrics.go

```diff
@@ -111,6 +111,7 @@ func init() {
 	prometheus.MustRegister(cacheCounter)
 	prometheus.MustRegister(timeoutDuration)
 	prometheus.MustRegister(cacheEntryAge)
+	prometheus.MustRegister(ghRequestWaitDurationHistVec)
```

Comment:
nit: can we put this together with the other `ghRequest...` register?

/ok-to-test

Reply:
Changed, also ordering in the file is changed (all gh metrics come first).